lib/Gocdb_Services/NotificationService.php

```diff
@@ -152,7 +154,8 @@ private function sendGocdbAdminsEmail($roleRequested, $requestingUser, $entityNa
             implode("\n", array(
                 'Dear GOCDB Admins,',
                 '',
-                '%1$s %2$s requested the "%3$s" role over %4$s, and there are no approving users available for the request.',
+                '%1$s %2$s requested the "%3$s" role over %4$s, and there are no 
+		approving users available for the request.',
```

Comment:
pretty certain this will insert a line break into the message. You want two separate strings concatenated
e.g. 'some ' . 'sentence'

Reply:
Thanks, changes made